src/vs/workbench/contrib/webview/electron-browser/webviewProtocols.ts

```diff
@@ -7,17 +7,17 @@ import { extname, sep } from 'vs/base/common/path';
 import { startsWith } from 'vs/base/common/strings';
 import { URI } from 'vs/base/common/uri';
 import { REMOTE_HOST_SCHEME } from 'vs/platform/remote/common/remoteHosts';
-import { ITextFileService } from 'vs/workbench/services/textfile/common/textfiles';
+import { IFileService } from 'vs/platform/files/common/files';
 
 export const enum WebviewProtocol {
 	CoreResource = 'vscode-core-resource',
 	VsCodeResource = 'vscode-resource',
 }
 
-function resolveContent(textFileService: ITextFileService, resource: URI, mime: string, callback: any): void {
-	textFileService.read(resource, { encoding: 'binary' }).then(contents => {
+function resolveContent(fileService: IFileService, resource: URI, mime: string, callback: any): void {
+	fileService.readFile(resource).then(contents => {
 		callback({
-			data: Buffer.from(contents.value, contents.encoding),
+			data: contents.value,
```

Comment:
This needs to be the real buffer value.  I'll merge this and make the fix since it otherwise looks good